spring-cloud-skipper-server/pom.xml

```diff
@@ -139,6 +133,11 @@
             <artifactId>spring-cloud-deployer-kubernetes</artifactId>
             <version>${spring-cloud-deployer-kubernetes.version}</version>
         </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
```

Comment:
Uh oh, please lord save us from guava! We need to find something else.

Reply:
Maybe we could build something from https://github.com/jgrapht/jgrapht. We're probably going to need more graph structures as that is what a complex deployment will look like.

Reply:
tinkerpop also - http://tinkerpop.apache.org/.  I think I'll be moving the `MapDifference`'s of each app into the `ReleaseDifference` instead of storing just a pretty-print of the differences, so a bit more structured.  The graph approach is prob best in the abstract, the right tool, "just" requires learning those tools....  ATM the shapes of the apps we need to support are very simple, deriving from what we want to do with SCDF, but can get more complex if/when skipper expands outside that use-case.